scipy/stats/tests/test_distributions.py

```diff
@@ -2922,6 +2922,22 @@ def test_sf_isf(self):
         xx = stats.invgamma.isf(y, 1)
         assert_allclose(x, xx, rtol=1.0)
 
+    @pytest.mark.parametrize("a, ref",
+                            [(100000000.0, -26.21208257605721),
+                            (10000000000.0, -33.11983786163935),
+                            (1e+20, -67.6586142566167),
+                            (1e+75, -257.6218844286255),
+                            (1e+100, -343.9688254159022)])
```

Comment:
Two very high values should be fine, we do not need five. What about we add also tests for small `a` if we are anyway adding tests for the entropy here?

Reply:
Ok, I've removed the middle 3 values. As for the small values, like I said in the PR for gengamma entropy, I'd only do that in a different PR because this one is only meant for large values.